docs/source/classroom/class_guide.md

```diff
@@ -62,54 +64,50 @@ The resources available in the courses/courses-gpu partitions can be queried wit
 sinfo -p courses-gpu  --Format=nodes,cpus,gres,statecompact
 :::
 
+:::{important}
+The compute resources for courses are shared across all courses each term. [We monitor their usage daily](../best-practices/clusterusage.md). **We send out email notifications to users who are idle on courses-gpu for one hour.** We highly recommend ending jobs when your work has finished as this frees up the resource for other students. When all students do this, it increases the availability of resources for everyone in courses. 
```

Comment:
Do we end the jobs in these cases? If so, we need to change the wording in this to indicate that after an hour, the job will be terminated.